GitHubTeams.ps1

```diff
@@ -691,219 +1629,328 @@ filter Set-GitHubTeam
     }
 }
 
-filter Rename-GitHubTeam
+filter Set-GitHubTeamProjectPermission
```

Comment:
The object being changed isn't the team.  In this case it's the project.  You're also not setting the list of supported teams (which would imply that this removes teams too), but rather only _adding_ a team with its permissions.  I'd suggest that this be a command in `GitHubProjects.ps1` and be called `Add-GitHubProjectTeam`.  Given that the endpoint can _both_ add _and_ update a single team's permission, it would be relevant to add an alias of `Set-GitHubProjectTeam` as well.